src/ui/components/VPNAnimatedRings.qml

```diff
@@ -64,6 +66,19 @@ Rectangle {
         property var ringXCenter: parent.width / 2
         property var ringYCenter: animatedRingsWrapper.yCenter
 
+        renderStrategy: Canvas.Threaded
+        renderTarget: Canvas.FramebufferObject
+
+        // Finds the Minimum Box that we need to repaint and marks this
+        // for painting
+        function makeDirty(){
```

Comment:
2nd thing i did, mostly for the background consumption: 
We can tell QT which part of the canvas needs to be updated, so it can check itself if a "dirty" part is visible and scheudle painting / or stop. 
I tested this on my macbook -> now gpu usage drops to <1% when i drag firefox over the window :)